runbotics/runbotics-desktop/src/action/microsoft/excel/excel.service.ts

```diff
@@ -142,7 +147,8 @@ export class ExcelService {
         // it uses the ASCII code of the letter to calculate the number
         // e.g A = 65, B = 66, C = 67, etc.
         // so we subtract 64 from the ASCII code to get the column number
-        return (column.length - 1) * 26 + (column.charCodeAt(column.length - 1) - 64);
+        const colnum = (column.length - 1) * 26 + (column.charCodeAt(column.length - 1) - 64);
+        return colnum;
```

Comment:
if we need to create a variable here then name it more clearly, avoid abbreviations

Reply:
no need for variable creation, forgot to remove it